src/emailcontactlistmodel.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2020 - 2021 Open Mobile Platform LLC.
+ *
+ * This program is licensed under the terms and conditions of the
+ * Apache License, version 2.0.  The full text of the Apache License is at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+#ifndef EMAILCONTACTLISTMODEL_H
+#define EMAILCONTACTLISTMODEL_H
+
+#include <QAbstractListModel>
+
+
+class Q_DECL_EXPORT EmailContactListModel : public QAbstractListModel
```

Comment:
This shouldn't need Q_DECL_EXPORT since it's internal? Same for the contact.

Reply:
done